packages/cicero-cli/lib/commands.js

```diff
@@ -496,7 +496,13 @@ class Commands {
     static archive(templatePath, target, outputPath, options) {
         return Commands.loadTemplate(templatePath, options)
             .then(async (template) => {
-                const archive = await template.toArchive(target);
+                const keyStore = {};
+                if (options.keyStore) {
```

Comment:
Please be consistent with casing of `keyStore`

Reply:
Sorry for that, I've corrected all of them: keystore, passphrase, timestamp.